eng/Packages.Data.props

```diff
@@ -63,6 +63,7 @@
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.6.1" />
```

Comment:
Another suggestion ;)

NuGet now supports this feature.  See https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions

Reply:
Thanks I filed https://github.com/Azure/azure-sdk-for-net/issues/15252 to investigate using that.